### PR TITLE
Added new `state export runtime` command.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-2773" title="DX-2773" target="_blank">TaskDX-2773</a>  As a user I should be able to programmatically receive the runtime dir
  </td></table>
  <br />

### CODE REVIEW

**Comment by mitchell-as** [comment]
The lone test failure is not due to this PR. It's one of the usual timeouts.

File: internal/runners/export/runtime.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+			return &ErrProjectNotFound{params.Path}
+		}
+	}
+	if proj == nil {
+		return rationalize.ErrNoProject
+	}
+
+	e.out.Notice(locale.Tr("export_runtime_statement", proj.NamespaceString(), proj.Dir()))
+
+	rt, err := runtime.SolveAndUpdate(e.auth, e.out, e.analytics, proj, nil, target.TriggerActivate, e.svcModel, e.cfg, runtime.OptMinimalUI)
```
Comment (MDrakos) [inline internal/runners/export/runtime.go:70]: We should probably add a new trigger for this. I don't believe this requires an update to the analytics doc but we should check to make sure.

File: internal/runners/export/runtime.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	execDir := setup.ExecDir(runtimeDir)
+
+	env, err := rt.Env(false, true)
+	if err != nil {
+		return errs.Wrap(err, "Could not get runtime environment")
+	}
+
+	tmpl, err := template.New("env").Parse(`{{- range $k, $v := .}}
+  - {{$k}}: {{$v}}
+{{- end}}`)
```
Comment (MDrakos) [inline internal/runners/export/runtime.go:86]: Nit: even though this is small we should consider adding this to the assets package to help with readability and be in line with other templates in our codebase.

**Review by MDrakos — CHANGES_REQUESTED** [review]
Looks really good, just two minor things.

File: internal/runners/export/runtime.go (hunk 1 above)
Comment (mitchell-as) [reply internal/runners/export/runtime.go]: Good catch, thanks!

**Review by MDrakos — APPROVED** [review]

**Pull request merged**